SU2_CFD/src/output_structure.cpp

```diff
@@ -4845,30 +4781,21 @@ void COutput::SetConvHistory_Body(ofstream *ConvHist_file,
           }
         }
         
-        
-        //        if (fluid_structure) {
-        //          Total_CFEA  = solver_container[ZONE_0][FinestMesh][FEA_SOL]->GetTotal_CFEA();
-        //        }
-        
-        if (output_1d) {
-          
-          /*--- Get area-averaged and flux-averaged values at the specified surface ---*/
-          
-          OneD_AvgStagPress = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_TotalPress();
-          OneD_AvgMach = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_Mach();
-          OneD_AvgTemp = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_Temp();
-          OneD_MassFlowRate = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_MassFlowRate();
-          
-          OneD_AvgPress = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_AvgPress();
-          OneD_AvgDensity = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_AvgDensity();
-          OneD_AvgVelocity = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_AvgVelocity();
-          OneD_AvgEnthalpy = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_AvgEnthalpy();
-        }
-        /*--- Get Mass Flow at the Monitored Markers ---*/
-        
-        
-        if (output_massflow) {
-          Total_Mdot = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetOneD_MassFlowRate();
+        /*--- Get flux-averaged values at the specified surface ---*/
+
+        if (output_surface) {
+
+          unsigned short iMarker_Analyze = 0;
```

Comment:
This should be improved... at least the solver should compute an average of all the markers to be analyzed